packages/server-wallet/src/wallet/wallet.ts

```diff
@@ -142,7 +140,7 @@ export class SingleThreadedWallet
 
     if (!valid) {
       errors.forEach(error =>
-        this.logger.error({error}, `Validation error occured ${error.message}`)
+        this.logger.error({error}, `Validation error occurred ${error.message}`)
```

Comment:
I think `err` is the magic keyword for pino, which triggers to use [the `pino-std-serializers/err` serializer](https://github.com/pinojs/pino-std-serializers#exportserrerror)

```suggestion
        this.logger.error({err: error}, `Validation error occurred ${error.message}`)
```